main.go

```diff
@@ -214,6 +214,61 @@ func main() {
 					return nil
 				},
 			},
+			{
+				Name:        "add git reminders",
```

Comment:
I feel like these command names are a bit long... seems weird to type 'canaveral [-options] add git reminders [args]'

Reply:
I guess you do have aliases. That makes it better

Reply:
Honestly, this is pretty stupid but I was under the impression that the name was the name of the functionality, not the name of the command. So like, the intended usage for a comamnd named [name] with alias [alias] would be `canaveral alias [args]` and the name was more used for me saying "Hey there's a bug with [name]". My bad lol, I can change these.